plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -37,19 +41,39 @@ public TestingMySqlServer(boolean globalTransactionEnable)
 
     public TestingMySqlServer(String dockerImageName, boolean globalTransactionEnable)
     {
-        super(dockerImageName);
-        withDatabaseName("tpch");
+        container = createContainer(dockerImageName, globalTransactionEnable);
+        container.start();
+        execute(format("GRANT ALL PRIVILEGES ON *.* TO '%s'", container.getUsername()), "root", container.getPassword());
+    }
+
+    private static MySQLContainer<?> createContainer(String dockerImageName, boolean globalTransactionEnable)
+    {
+        MySQLContainer<?> container = new MySQLContainer<>(dockerImageName);
+        container = container.withDatabaseName("tpch");
         if (globalTransactionEnable) {
-            withCommand("--gtid-mode=ON", "--enforce-gtid-consistency=ON");
+            container = container.withCommand("--gtid-mode=ON", "--enforce-gtid-consistency=ON");
         }
-        start();
-        execute(format("GRANT ALL PRIVILEGES ON *.* TO '%s'", getUsername()), "root", getPassword());
+        return container;
+    }
+
```

Comment:
consider the same ordering for methods (and already existed methods too) in both `Testing classes`.